src/annotator/sidebar.js

```diff
@@ -56,17 +56,20 @@ export default class Sidebar {
    * @param {HTMLElement} element
    * @param {import('./util/emitter').EventBus} eventBus -
    *   Enables communication between components sharing the same eventBus
+   * @param {MessagePort} hostPort -
+   *   Enables communication with the sidebar frame
```

Comment:
It might be useful to add some additional context here. eg. "Host port for the host <-> sidebar communication channel. The sidebar app will request its side of the channel when it starts."